packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -155,3 +159,113 @@ export interface AppWithRepository extends Application {
   component(component: Class<{}>): void;
   mountComponentRepository(component: Class<{}>): void;
 }
+
+/**
+ * A dummy class created to generate the tsdoc for the members in repository
+ * mixin. Please don't use it.
+ *
+ * The members are implemented in function
+ * <a href="#RepositoryMixin">RepositoryMixin</a>
+ */
+export class RepositoryMixinDoc {
```

Comment:
Is it necessary to `export` this class? I am concerned about pollution of our API, can we keep the class internal to this source file?

Reply:
@bajtos I am afraid yes, `storng-docs` only detects exported item, see [code](https://github.com/strongloop/strong-docs/blob/master/src/ts-parser.ts#L89)

The dummy class throws error in its constructor to prevent people from using it.